handshake/connection_parameters_manager.go

```diff
@@ -242,7 +246,7 @@ func (h *connectionParametersManager) GetMaxIncomingStreams() uint32 {
 	h.mutex.RLock()
 	defer h.mutex.RUnlock()
 
-	maxStreams := protocol.MaxIncomingDynamicStreamsPerConnection
+	maxStreams := h.configuredMaxIncomingDynamicStreamsPerConnection
```

Comment:
That does seem logical.
I'm also wondering whether [this](https://github.com/lucas-clemente/quic-go/blob/master/handshake/connection_parameters_manager.go#L84) and [this](https://github.com/lucas-clemente/quic-go/blob/master/handshake/connection_parameters_manager.go#L88) should not be based on protocol.MaxStreamsPerConnection but on configuredMaxIncomingDynamicStreamsPerConnection